File: bubble_sort.py
import random

def bubble_sort(arr):
	swapped = True

	while swapped:
		swapped = False
		for i in range(len(arr) - 1):
			if arr[i] > arr[i+1]:
				swapped = True
				arr[i], arr[i+1] = arr[i+1], arr[i]

	return arr

if __name__ == "__main__":
	arr = []
	
	for i in range(10):
		arr.append(random.randint(1, 20))

	print(f"Unsorted Array: {arr}")
	print(f"Sorted by Bubble sort: {bubble_sort(arr)}")

File: js/bubble_sort.js
function bubbleSort(arr) {
	let swapped = true;

	while (swapped) {
		swapped = false;

		for (let i = 0; i < arr.length - 1; i++) {
			if (arr[i] > arr[i + 1]){
				[arr[i], arr[i + 1]] = [arr[i + 1], arr[i]];
				swapped = true
			}
		}
	}

	return arr;
}

let unsorted = [5, 2, 4, 6, 1, 3];
console.log(unsorted);
console.log(bubbleSort(unsorted));